heltour/tournament/api.py

```diff
@@ -386,6 +386,12 @@ def set_availability(request):
     
     if season_player.card_color == "red":
         return JsonResponse({'updated': 0, 'error': 'player_red_card'})
+    
+    if season_player.has_scheduled_game_in_round(round_):
+        return JsonResponse({'updated': 0, 'error': 'scheduled_game'})
+    
+    if has_scheduled_game:
```

Comment:
Where is this variable set? Why is the error for this clause different from the error in the one above?